### ISSUE TITLE
Plans 2023: Add tax exclusion copy to /pricing page

### ISSUE THREAD

**Issue by jeyip** [Automattic/wp-calypso#78296]
### Details

We don't say/show anywhere on the [pricing page](https://wordpress.com/pricing/) that the prices shown are tax exclusive;

<img width="1496" alt="Screenshot 2023-06-15 at 5 38 20 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/9f6f464b-16ad-4f63-a115-9bdb77eb0da6">

We do on the plan page (when looked at a site in the dashboard);

<img width="1496" alt="Screenshot 2023-06-15 at 5 38 38 PM" src="https://github.com/Automattic/wp-calypso/assets/5414230/8b7b4323-6e46-4c4e-b717-ac654128d5a2">

We should add tax exclusion copy for relevant countries

### Related

p7DVsv-gc4-p2#comment-46307

**Comment by southp** [comment]
@Automattic/martech-autobots Is this something you can handle? Also, fwiw it can be shown in better precision using the backend data discussed here: https://github.com/Automattic/martech/issues/1820#issue-1745068250 :)

**Comment by niranjan-uma-shankar** [comment]
Sorry for the late reply. I noticed a while back that our pricing page needs parity with the logged-in pages in https://github.com/Automattic/martech/issues/1770.  We'll roll up this update as part of that task.

**Comment by jeyip** [comment]
Thanks @niranjan-uma-shankar!

**Comment by niranjan-uma-shankar** [comment]
### Update

This issue needs design input to move forward. A quick summary is that we show "Excl taxes" in Calypso plan pages but only on the annual tab. It is missing on the monthly tab in Calypso, and entirely missing on all landing pages (both in annual and monthly tabs).

Questions to move forward:

* At a zoomed out level, I would first question whether we even need to show "excl taxes" copy. This implementation came as a result of the discussions in pdgrnI-27y-p2 and. However, we display prices in several other places in dotcom without "excl taxes" copy - upsells, emails, plugins pages, themes pages, editor page, and so on and we don't display "excl taxes" anywhere else. IMO we should either aim for consistency with showing it everywhere (which would be a large'ish effort to cover all of dotcom where we display prices) or we postpone it for later. If we're choosing the latter, I suggest removing "Excl taxes" entirely and get stakeholder approval before doing so.
* If we decide that we want to show "Excl taxes" only on the pricing grid for now, then pcNC1U-Ua-p2/#excl-taxes-copy summarises the remaining work. Currently needs design input to move forward. I'd recommend also considering a holistic solution to includes the toolip suggestion given pcNC1U-Ua-p2/#special-discount-currencies-inr-mxn-php.

**Comment by DavidRothstein** [comment]
Came up here also: pNPgK-6LT-p2

>It is missing on the monthly tab in Calypso, and entirely missing on all landing pages (both in annual and monthly tabs).

Just want to note that it's also missing on https://wordpress.com/me/purchases (and the individual purchase management pages linked therein) so if the decision is made to try to show it consistently everywhere, those are prominent places where a renewal price is displayed that would make sense to update also.

I'm adding this to the Shilling Team's board for discussion also.

**Comment by michaeldcain** [comment]
We'll handle the purchase management changes with the following issue: https://github.com/Automattic/payments-shilling/issues/2184

**Comment by saygunnyc** [comment]
This is already on the main flow. So, it would be a matter of carrying it to other instances. Can we please fix the "Excl. Taxes" copy as lowercase "excl. taxes" everywhere? It's been mentioned a few times on other PRs.


![Image](https://github.com/Automattic/wp-calypso/assets/78876323/8f89bd2f-c8a4-43ef-94d8-dee79809dc84)

**Comment by niranjan-uma-shankar** [comment]
@saygunnyc I'd like to hear your thoughts on [this comment](https://github.com/Automattic/wp-calypso/issues/78296#issuecomment-1807467581) which summarises the pending tasks. If we're keen to add the "excl taxes" copy, then a holistic solution will involve looking at all the places where it needs to be added. We can take this up separately but would involve other teams, just dropping it on your radar to consider.

Coming to the pricing grid, pcNC1U-Ox-p2 describes the issue further - the blocker here is on how we want to handle the copy in the monthly plans tab which is missing "excl taxes" in all instances. I would propose something like the one below, I need a 👍 to implement this, or if you have a different suggestion please let me know.

### What we have in production (monthly tab)

<img width="243" alt="Screenshot 2023-11-23 at 2 38 31 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/a27b2a8b-2625-42c8-9127-9c00e22f983d">

### Proposed (monthly tab)

<img width="251" alt="Screenshot 2023-11-23 at 2 38 18 PM" src="https://github.com/Automattic/wp-calypso/assets/1269602/ec4c9a7c-2732-4348-818b-5fab512acaec">

**Comment by niranjan-uma-shankar** [comment]
> Can we please fix the "Excl. Taxes" copy as lowercase "excl. taxes" everywhere? It's been mentioned a few times on other PRs.

@saygunnyc  Fixed in https://github.com/Automattic/wp-calypso/pull/84466

**Comment by lucasmendes-design** [comment]
@ianstewart I think we still need this.

**Comment by saygunnyc** [comment]
>  Coming to the pricing grid, pcNC1U-Ox-p2 describes the issue further -
... The text below the plan price is talking about the savings here

I can't find the exact PR or issue at the moment (maybe @jeyip or @southp would recall); however, I previously mentioned that's not the place to mention savings and is the place where we mention billing cycle or tax information. We already have a guideline with crossed out prices to display savings.

<img width="1442" alt="Screenshot 2023-11-27 at 5 27 03 PM" src="https://github.com/Automattic/wp-calypso/assets/78876323/2a80ab3a-dfbb-448a-bb78-c87720030cd5">


We also were planning to move forward with this dropdown where we can display the discount amount and more payment terms instead of the payment term toggle. Not sure where we are with it.

<img width="182" alt="Screenshot 2023-11-27 at 5 30 52 PM" src="https://github.com/Automattic/wp-calypso/assets/78876323/0f658762-9152-4264-afc9-0dd7db9710cb">
<img width="189" alt="Screenshot 2023-11-27 at 5 30 57 PM" src="https://github.com/Automattic/wp-calypso/assets/78876323/97f52ae2-c87e-4b1e-94e2-404ce82bbefe">


@niranjan-uma-shankar To unblock, https://github.com/Automattic/wp-calypso/issues/78296#issuecomment-1824184661 could be a quick temporary fix in the meantime.

cc @vinimotaa

**Comment by southp** [comment]
> We also were planning to move forward with this dropdown where we can display the discount amount and more payment terms instead of the payment term toggle. Not sure where we are with it.

@saygunnyc It's on the list of GF signup. According to @isatuncman-auto 's latest update(p1701294813968819-slack-C059R8S4ALT) , we will priortize it next.

**Comment by niranjan-uma-shankar** [comment]
> @niranjan-uma-shankar To unblock, https://github.com/Automattic/wp-calypso/issues/78296#issuecomment-1824184661 could be a quick temporary fix in the meantime.

Noted, thanks.

**Comment by niranjan-uma-shankar** [comment]
> It's on the list of GF signup. According to @isatuncman-auto 's latest update(p1701294813968819-slack-C059R8S4ALT) , we will priortize it next.

@southp Will that experiment run on `/start/onboarding-pm` or in the main `/start` flow? Since the pm flow and main flow get very different audience types - the pm flow gets people with a stronger intent since they have responded to an ad - do we need to run a test once again on the main flow if the pm flow experiments wins?

**Comment by southp** [comment]
@niranjan-uma-shankar The other day @victorespigares has posted a great PT summarizing its spec: peP6yB-1F2-p2. It's true that the audience has very different intention, but I think it's a general UX exploration for better utilizing our limited space to better communiate our offer and the benefit of paying in longer terms. Thus, we will run it on both the paid media flow and the main onboarding flow, and we won't need to run the experiment again if the experiment wins :)

**Comment by southp** [comment]
The main issue tracked here is alread resolved by @Automattic/martech-autobots already, so I'm closing this.
